[2024-11-18T12:15:29.121+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T12:15:29.229+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_weather_data manual__2024-11-18T12:15:18.127578+00:00 [queued]>
[2024-11-18T12:15:29.261+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_weather_data manual__2024-11-18T12:15:18.127578+00:00 [queued]>
[2024-11-18T12:15:29.262+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-11-18T12:15:29.299+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2024-11-18 12:15:18.127578+00:00
[2024-11-18T12:15:29.358+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2293) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-18T12:15:29.365+0000] {standard_task_runner.py:72} INFO - Started process 2294 to run task
[2024-11-18T12:15:29.355+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_pipeline', 'transform_weather_data', 'manual__2024-11-18T12:15:18.127578+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmpbb626uzz']
[2024-11-18T12:15:29.370+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask transform_weather_data
[2024-11-18T12:15:29.671+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_pipeline.transform_weather_data manual__2024-11-18T12:15:18.127578+00:00 [running]> on host 527ca48b98a5
[2024-11-18T12:15:30.288+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='naphat' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:15:18.127578+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T12:15:18.127578+00:00'
[2024-11-18T12:15:30.304+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T12:15:30.344+0000] {python.py:240} INFO - Done. Returned value was: {'Time of Record': '2024-11-18 12:06:38', 'Country': 'TH', 'City': 'Bangkok', 'Latitude': 13.75, 'Longitude': 100.5167, 'Current Weather': 'Clouds', 'Temperature (F)': 84.29000000000002, 'Feels Like (F)': 87.458, 'Minimun Temp (F)': 82.90400000000005, 'Maximum Temp (F)': 85.892, 'Pressure': 1009, 'Humidty': 58, 'Wind Speed': 1.98, 'Sunrise (Local Time)': '2024-11-17 23:19:15', 'Sunset (Local Time)': '2024-11-18 10:47:17'}
[2024-11-18T12:15:30.723+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T12:15:30.724+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=transform_weather_data, run_id=manual__2024-11-18T12:15:18.127578+00:00, execution_date=20241118T121518, start_date=20241118T121529, end_date=20241118T121530
[2024-11-18T12:15:30.819+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T12:15:31.350+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-18T12:15:31.371+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
